splunk.json

```diff
@@ -20,7 +20,7 @@
     ],
     "logo": "logo_splunk.svg",
     "logo_dark": "logo_splunk_dark.svg",
-    "license": "Copyright (c) 2014-2021 Splunk Inc.",
+    "license": "Copyright (c) 2014-2022 Splunk Inc.",
```

Comment:
Your changes look great. For future reference, It should be sufficient to just update the license year here.
Another job will use the updated year here to update the license years in other locations.

Reply:
okay thanks for the information